.github/workflows/buf.yaml

```
@@ -8,12 +8,21 @@ jobs:
       - uses: actions/checkout@v3
       - uses: bufbuild/buf-setup-action@v1
       - uses: bufbuild/buf-lint-action@v1.0.3
+  breaking:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: bufbuild/buf-setup-action@v1
+      - uses: bufbuild/buf-breaking-action@v1.1.2
```

```suggestion
      - uses: bufbuild/buf-breaking-action@v1
```

In general I think we should update all of our action versions to `v1`.

---

I'd agree